src/polus/plugins/_plugins/classes/plugin_classes.py

```diff
@@ -38,32 +41,32 @@
 Paths and Fields
 """
 # Location to store any discovered plugin manifests
-PLUGIN_DIR = pathlib.Path(__file__).parent.parent.joinpath("manifests")
+_PLUGIN_DIR = pathlib.Path(__file__).parent.parent.joinpath("manifests")
 
 
 def load_config(config: typing.Union[dict, pathlib.Path]):
     """Load configured plugin from config file/dict."""
     if isinstance(config, pathlib.Path):
-        with open(config) as fr:
-            m = json.load(fr)
+        with open(config, encoding="utf-8") as file:
+            manifest_ = json.load(file)
     elif isinstance(config, dict):
-        m = config
+        manifest_ = config
     else:
         raise TypeError("config must be a dict or a path")
-    _io = m["_io_keys"]
-    cl = m["class"]
-    m.pop("class", None)
-    if cl == "Compute":
-        pl = ComputePlugin(_uuid=False, **m)
-    elif cl == "WIPP":
-        pl = Plugin(_uuid=False, **m)
+    _io = manifest_["_io_keys"]
```

Comment:
always prefer descriptive names to short obscure variable names such pl_ or cl_

Reply:
Solved